changelog.d/20250710_132341_yadudoc1729_add_get_endpoint_options.rst

```diff
@@ -0,0 +1,5 @@
+New Functionality
+^^^^^^^^^^^^^^^^^
+
+- Added a new ``role`` kwarg to ``Client.get_endpoints`` method that enables querying for endpoint that
+  are not owned by the user such as MEPs. Valid options for ``role`` are "owner" and "any".
```

Comment:
- `s/Added a new/Add a new/`
- `s/kwarg to/kwarg to the/`
- `s/querying for endpoint/querying for endpoints/`

Might consider making `owner` and `any` as monotype ``` ``owner`` ``` and ``` ``any`` ```, rather that quoted words, to make clear the values.

Reply:
Done.